jdk-annotations/test-annotated/src/main/java/com/uber/nullaway/jdkannotations/ReturnAnnotation.java

```diff
@@ -76,4 +78,8 @@ public static Object getNewObjectIfNull(@Nullable Object object) {
       returnNullableGenericContainingNullable() {
     return null;
   }
+
+  public static @Nullable List<? super String> getList(@NonNull Integer i, @Nullable Character c) {
```

Comment:
Since this class is `@NullMarked` you shouldn't need the `@NonNull` annotation on `i`